Directory.Build.props

```diff
@@ -57,6 +101,9 @@
     <SourceDir>$(ProjectDir)src\</SourceDir>
     <BinDir>$(ProjectDir)artifacts\bin\</BinDir>
 
+    <!-- Need to try and keep the same logic as the native builds as we need this for packaging -->
+    <NativeBinDir>$(BinDir)native/$(BuildConfiguration)</NativeBinDir>
```

Comment:
Not 100% sure if you really want to rely on $(BinDir). Wes grouped it to other obsolete properties which should be removed when buildtools is gone.

Reply:
Arcade defines: `ArtifactsBinDir` https://github.com/dotnet/arcade/blob/a991a1d46cf6e5ed7dde93309217306f6a42c136/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L68

But the SDK props are not imported until ~ line 200

Reply:
This isn't some new dependency, it's just reordering an existing property.  I'm ok with it moving to a different path / property but I don't think I should be doing that in this PR.

Reply:
Agreed, let's defer it till we remove buildtools entirely.